helm/cas-registration/templates/backend/secret.yaml

```diff
@@ -9,7 +9,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ include "cas-registration.fullname" . }}-backend
+  name: {{ .Release.Name }}-backend
```

Comment:
we need to change the `deployment.yaml` file instead to use the helm named template